packages/components/psammead-bulletin/src/index.jsx

```diff
@@ -67,52 +45,86 @@ const TVBulletinWrapper = styled.div`
   }
 `;
 
-const TVImageWrapper = styled.div`
+// This is needed to get around the issue of IE11 not supporting
+// nested media queries
+const headlineTypography = script => {
+  const fontSize = script.greatPrimer.groupD.fontSize / 16;
+  const lineHeight = script.greatPrimer.groupD.lineHeight / 16;
+
+  return css`
+    font-size: ${fontSize}rem;
+    line-height: ${lineHeight}rem;
+  `;
+};
+
+const headingStyles = css`
+  ${({ service }) => service && getSerifMedium(service)}
+  ${({ script }) => script && getPica(script)}
+  color: ${C_EBON};
+  margin: 0; /* Reset */
+  padding: ${GEL_SPACING};
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
```

Comment:
We will need to add another media query for groupB.
If you update the `headlineTypography` function to also take in a group string, e.g. 'groupD', we could have this:

```
@media (min-width: ${GEL_GROUP_B_MIN_WIDTH}) and (max-width: ${GEL_GROUP_B_MAX_WIDTH}) {
  ${({ script }) => script && headlineTypography(script, 'groupB')}
}
@media (min-width: ${GEL_GROUP_D_MIN_WIDTH}) {
  ${({ script }) => script && headlineTypography(script, 'groupD')}
}

Reply:
I checked on browserstack and it is all working well.